src/test/java/org/rulez/demokracia/pdengine/BeatTableGetPairTest.java

```diff
@@ -0,0 +1,121 @@
+package org.rulez.demokracia.pdengine;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.rulez.demokracia.pdengine.BeatTable.Direction;
+import org.rulez.demokracia.pdengine.annotations.TestedBehaviour;
+import org.rulez.demokracia.pdengine.annotations.TestedFeature;
+import org.rulez.demokracia.pdengine.annotations.TestedOperation;
+import org.rulez.demokracia.pdengine.dataobjects.Pair;
+import org.rulez.demokracia.pdengine.exception.IsNullException;
+import org.rulez.demokracia.pdengine.exception.ReportedException;
+import org.rulez.demokracia.pdengine.testhelpers.CreatedDefaultChoice;
+
+public class BeatTableGetPairTest extends CreatedDefaultChoice {
```

Comment:
I doubt we need that many test to introduce functionality of calling a single function with two arguments.